src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBookStub.json

```diff
@@ -1,46 +0,0 @@
-{
-  "_comment": "AddressBook save file which contains the same Person values as in TypicalPersons#getTypicalAddressBook()",
-  "persons" : [ {
-    "name" : "Alice Pauline",
-    "email" : "alice@example.com",
-    "module": "CS2103T",
-    "assignments" : [  ],
-    "tagged" : [ "friends" ]
-  }, {
-    "name" : "Benson Meier",
-    "email" : "johnd@example.com",
-    "module": "CS1101S",
-    "assignments" : ["Assignment 2|15 Oct 2021, 01:00 PM|PENDING", "Homework 1|13 Sep 2021, 11:59 PM|LATE" ],
-    "tagged" : [ "owesMoney", "friends" ]
-  }, {
-    "name" : "Carl Kurz",
-    "email" : "heinz@example.com",
-    "module": "IS2101",
-    "assignments" : ["iP|17 Sep 2021, 11:59 PM|COMPLETED", "Assignment 2|15 Oct 2021, 01:00 PM|PENDING" ],
-    "tagged" : [ ]
-  }, {
-    "name" : "Daniel Meier",
-    "email" : "cornelia@example.com",
-    "module": "CM1102",
-    "assignments" : [ "Homework 1|13 Sep 2021, 11:59 PM|LATE" ],
-    "tagged" : [ "friends" ]
-  }, {
-    "name" : "Elle Meyer",
-    "email" : "werner@example.com",
-    "module": "GER1000",
-    "assignments" : [ "Tutorial 2|16 Sep 2021, 11:59 PM|COMPLETED" ],
-    "tagged" : [ ]
-  }, {
-    "name" : "Fiona Kunz",
-    "email" : "lydia@example.com",
-    "module": "GET1031",
-    "assignments" : [ "Assignment 1|10 Sep 2021, 11:59 PM|COMPLETED" ],
-    "tagged" : [ ]
-  }, {
-    "name" : "George Best",
-    "email" : "anna@example.com",
-    "module": "MA1101R",
-    "assignments" : [ "Lab 1|20 Sep 2021, 11:59 PM|COMPLETED" ],
-    "tagged" : [ ]
-  } ]
```

Comment:
if it's never used,free to remove, if some other tests coming later needs it, it's harmless to keep it